src/lib/utils/FileFactory.ts

```diff
@@ -28,7 +28,7 @@ export class FileFactory {
 
     public addFile<T extends BscFile>(destPath: string, contents: string) {
         try {
-            return this.program.setFile<T>({ src: path.resolve(destPath), dest: destPath }, contents);
+            return this.program.setFile<T>(destPath, contents);
```

Comment:
Yeah, this is good. BrighterScript will handle auto-resolving those paths, so `path.resolve()` would be wrong most of the time.